frontend/src/concepts/pipelines/content/import/ImportPipelineSplitButton.tsx

```diff
@@ -14,16 +16,23 @@ import PipelineVersionImportModal from '~/concepts/pipelines/content/import/Pipe
 type ImportPipelineSplitButtonProps = {
   onImportPipeline?: (pipeline: PipelineKF) => void;
   onImportPipelineVersion?: (pipelineVersion: PipelineVersionKF) => void;
+  variant?: MenuToggleProps['variant'];
+  disableAll?: boolean;
```

Comment:
hmmm... why "All" and not just disable? If the top button is disabled the dropdown makes no sense.

Reply:
Yeah I just didn't know how to do wording... 🤷‍♂️